misc/ls-extensions/modules/json-to-record-converter/src/main/java/io/ballerina/converters/JsonToRecordConverter.java

```diff
@@ -0,0 +1,397 @@
+                    ArraySchema arraySchema = (ArraySchema) schemaValue;
+                    Token openSBracketToken = AbstractNodeFactory.createToken(SyntaxKind.OPEN_BRACKET_TOKEN);
+                    Token closeSBracketToken = AbstractNodeFactory.createToken(SyntaxKind.CLOSE_BRACKET_TOKEN);
+                    IdentifierToken fieldName =
+                            AbstractNodeFactory.createIdentifierToken(escapeIdentifier(
+                                    schema.getKey().trim().toLowerCase(Locale.ENGLISH)) + "list");
+                    Token semicolonToken = AbstractNodeFactory.createIdentifierToken(";");
+                    TypeDescriptorNode fieldTypeName;
+                    if (arraySchema.getItems() != null) {
+                        fieldTypeName = extractOpenApiSchema(arraySchema.getItems(), schema.getKey(),
+                                typeDefinitionNodeList);
+                    } else {
+                        Token type =
+                                AbstractNodeFactory.createToken(SyntaxKind.STRING_KEYWORD);
+                        fieldTypeName =  NodeFactory.createBuiltinSimpleNameReferenceNode(null, type);
+                    }
+                    ArrayTypeDescriptorNode arrayField =
+                            NodeFactory.createArrayTypeDescriptorNode(fieldTypeName, openSBracketToken,
+                                    null, closeSBracketToken);
+                    RecordFieldNode recordFieldNode = NodeFactory.createRecordFieldNode(null,
+                            null, arrayField, fieldName, null, semicolonToken);
+                    NodeList<Node> fieldNodes = AbstractNodeFactory.createNodeList(recordFieldNode);
+                    Token bodyEndDelimiter = AbstractNodeFactory.createToken(SyntaxKind.CLOSE_BRACE_TOKEN);
+                    RecordTypeDescriptorNode recordTypeDescriptorNode =
+                            NodeFactory.createRecordTypeDescriptorNode(recordKeyWord, bodyStartDelimiter,
+                                    fieldNodes, null, bodyEndDelimiter);
+                    Token semicolon = AbstractNodeFactory.createToken(SyntaxKind.SEMICOLON_TOKEN);
+                    TypeDefinitionNode typeDefinitionNode = NodeFactory.createTypeDefinitionNode(null,
+                            null, typeKeyWord, typeName, recordTypeDescriptorNode, semicolon);
+                    typeDefinitionNodeList.add(typeDefinitionNode);
+                }
+            }
+        }
+
+
+        return new ArrayList<>(typeDefinitionNodeList);
+    }
+
+    /**
+     * Method for generating record fields with given schema properties.
+     *
+     * @param field schema entry of the field
+     * @param recordFieldList record field list
+     * @param required is it a required field.
+     * @throws JsonToRecordConverterException in case of bad schema entries
+     */
+    private static void addRecordFields(List<String> required, List<Node> recordFieldList,
```

Comment:
The Javadoc comment needs to be updated here.

Reply:
Fixed with c4bd7b5